qe_metrics/libs/orm_database.py

```diff
@@ -0,0 +1,71 @@
+from pony import orm
+from datetime import date, datetime
+
+from jira import Issue
+from simple_logger.logger import get_logger
+from typing import List, Any
+
+
+DB_CONNECTION: orm.Database = orm.Database()
+
+
+class JiraIssuesEntity(DB_CONNECTION.Entity):
+    """
+    A class to represent the JiraIssues table in the database.
+    """
+
+    _table_ = "jiraissues"
+
+    logger = get_logger(name=__module__)
+
+    id = orm.PrimaryKey(int, auto=True)
+    product = orm.Required("ProductsEntity")
+    issue_key = orm.Required(str)
+    title = orm.Required(str)
+    url = orm.Required(str)
+    project = orm.Required(str)
+    severity = orm.Required(str)
+    status = orm.Required(str)
+    customer_escaped = orm.Required(bool)
+    date_created = orm.Required(date)
+    last_updated = orm.Required(date)
+    date_record_modified = orm.Required(datetime, default=datetime.now)
```

Comment:
why is this needed?

Reply:
https://github.com/RedHatQE/qe-metrics/pull/59#discussion_r1542911790